f5_sphinx_theme/static/css/f5-theme.css

```diff
@@ -949,47 +1031,3 @@ table.docutils.footnote, table.docutils.footnote > tbody, table.docutils.footnot
     padding-left: 0;
   }
 }
```

Comment:
We can't just take out all of the version drop-down elements. This needs to be tested on a doc set that's using the version nav and the elements should be updated as needed to fit with the new styles. 
(f5-rest in GitLab)

Reply:
My fault. CSS we were working with was missing that section for some reason. You can see version selector in action here: http://10.145.104.168:8080/products/iapp/iapp-lx/tmos-14_0/  (It's a bit broken but the top-level page shows what you need to see)